docs/UserGuide.md

```diff
@@ -185,35 +185,62 @@ Examples:
 *  `delete appointment 7` Deletes the `appointment` at index `7`.
 *  `delete property 7` Deletes the `property` at index `7`.
 
-### Filtering: `find`
+### Searching properties: `find property`
 
-Finds properties or appointments that matches the criterion provided.
+Finds properties that match the criterion provided.
 
 Formats:
-* `find appointment [keywords] [option...]`
-* `find property [keywords] [option...]`
+* `find property [KEYWORD] [OPTION...]`
```

Comment:
I think it will be better if the `...` are after the square brackets.
E.g. `find property [KEYWORD] [OPTION]...`